modules/channel-web/src/views/lite/main.tsx

```diff
@@ -158,6 +160,25 @@ class Web extends React.Component<MainProps> {
     this.handleResetUnreadCount()
   }
 
+  /*
+    Call example:
+
+    const payload = bp.RealTimePayload.forVisitor(event.target, 'webchat.data', {
+      payload: {
+        currentUILanguage: 'ar'
+      }
+    })
+    bp.realtime.sendPayload(payload)
+  */
+  handleDataMessage = event => {
+    console.log(event.payload)
+    if (!event || !event.payload) return
+    const { currentUILanguage } = event.payload
+    if (!currentUILanguage) return
+
```

Comment:
Please use brackets at all time, even for one-line returns. 

Also, couldn't we just use `event.payload.language` instead ? "currentUILanguage" just seems too long for nothing, and the casing must be perfect or it will not work